addons/l10n_gcc_invoice_stock_account/views/report_invoice.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="arabic_english_invoice_with_snln" inherit_id="l10n_gcc_invoice.arabic_english_invoice">
+        <xpath expr="//div[@id='total']" position="after">
+            <t t-set="lot_values" t-value="o._get_invoiced_lot_values()"/>
+            <t t-if="lot_values">
+                <br/>
+                <table groups="stock_account.group_lot_on_invoice" class="table table-sm" style="width: 50%;" name="invoice_snln_table">
+                    <thead>
+                        <tr>
+                            <th><span>Product</span></th>
+                            <th class="text-right"><span>Quantity</span></th>
```

Comment:
Needs to be translated in Arabic then as well to be dual language.  I wonder if this change is worth the effort.  @akr-odoo @bakboka

Reply:
@jco-odoo the translation itself is 3 words, I can help with that if someone provides the XML edits

Reply:
I can even just put the Arabic word next to the English word

Reply:
If that does the trick and looks ok, why not.